### PR TITLE
[Monit-27533] Remove deprecated classes post grpc and otel dep updates

### PR DESCRIPTION
**Pull request by deosu**
- Change class InstrumentationLibraryMetrics to ScopeMetrics.
- Change class InstrumentationLibrarySpans to ScopeSpans.

### CODE REVIEW

**Review by keep94 — APPROVED** [review]
LGTM

File: proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java [hunk 1]
```diff
@@ -332,15 +332,15 @@ static List<Annotation> annotationsFromAttributes(List<KeyValue> attributesList)
   }
 
   @VisibleForTesting
-  static List<Annotation> annotationsFromInstrumentationLibrary(InstrumentationLibrary iLibrary) {
-    if (iLibrary == null || iLibrary.getName().isEmpty()) return Collections.emptyList();
+  static List<Annotation> annotationsFromInstrumentationLibrary(InstrumentationScope scope) {
```
Comment (oppegard) [inline proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java:335]: Rename to `annotationsFromInstrumentationScope()`.

File: proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtilsTest.java [hunk 2]
```diff
@@ -432,11 +427,11 @@ public void transformSpanGivesSpanAttributesHigherPrecedenceThanResourceAttribut
 
   @Test
   public void transformSpanHandlesInstrumentationLibrary() {
```
Comment (oppegard) [inline proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtilsTest.java:429]: Rename test to `transformSpanHandlesInstrumentationScope()`.

File: proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java (hunk 1 above)
Comment (deosu) [reply proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java]: Done

File: proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtilsTest.java (hunk 2 above)
Comment (deosu) [reply proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtilsTest.java]: Done

**Review by keep94 — APPROVED** [review]
LGTM

**Review by oppegard — APPROVED** [review]
LGTM - please rebase away the first two commits since they've already been squash-merged into `otel-metrics`.

**Pull request closed without merging**